WPFHospitalEditor/AdditionalInformation.xaml.cs

```diff
@@ -25,12 +29,20 @@ public partial class AdditionalInformation : Window
         private List<TextBox> textBoxes = new List<TextBox>();
         private TextBox name = new TextBox();
         private Grid DynamicGrid = new Grid();
+        private int floor;
+        private Building building;
+        private string[] descriptionParts;
 
-        public AdditionalInformation(MapObject mapObject)
+        MapObjectController mapObjectController = new MapObjectController(new MapObjectService(new MapObjectRepository(new FileRepository(AllMapObjects.MAPOBJECT_PATH))));
```

Comment:
Do you use this controller only in this class?

Reply:
Yes.